src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitServiceTest.kt

```diff
@@ -152,45 +150,6 @@ internal class VisitServiceTest {
     visitors.add(VisitVisitor(visit = this, person = Person(-7L, "First", "Last")))
   }
 
-  private val defaultLocation = AgencyLocation(id = "SWI", description = "Swindon")
```

Comment:
don't need these, since Mocks changed to echo back what was supplied for the rep `save` method